findContact/FindContactModal.js

```diff
@@ -1,34 +1,146 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get, noop } from 'lodash';
 
+import { SearchAndSort, makeQueryFunction } from '@folio/stripes/smart-components';
 import {
   Modal,
+  Button,
+  Icon,
 } from '@folio/stripes/components';
+import { transformCategoryIdsToLables } from '@folio/organizations/src/common/utils/category';
+import { categoriesResource } from '@folio/organizations/src/common/resources';
+import { CONTACTS_API } from '@folio/organizations/src/common/constants';
+
+import packageInfo from '../package';
+import css from './FindContactModal.css';
+
+const INITIAL_RESULT_COUNT = 30;
+const RESULT_COUNT_INCREMENT = 30;
+
+const filterConfig = [];
+const visibleColumns = ['status', 'name', 'categories'];
+const columnMapping = {
+  status: <FormattedMessage id="ui-plugin-find-contact.list.status" />,
+  name: <FormattedMessage id="ui-plugin-find-contact.list.name" />,
+  categories: <FormattedMessage id="ui-plugin-find-contact.list.categories" />,
+};
 
 class FindContactModal extends React.Component {
+  static manifest = Object.freeze({
+    query: {
+      initialValue: {
+        query: '',
+        filters: ''
+      },
+    },
+    records: {
+      type: 'okapi',
+      records: 'contacts',
+      path: CONTACTS_API,
+      clear: true,
+      recordsRequired: '%{resultCount}',
+      perRequest: RESULT_COUNT_INCREMENT,
+      GET: {
+        params: {
+          query: makeQueryFunction(
+            'cql.allRecords=1',
+            '(firstName="*%{query.query}*" or lastName="*%{query.query}*")',
+            {},
+            filterConfig,
+          ),
+        },
+        staticFallback: { params: {} },
+      },
+    },
+    resultCount: { initialValue: INITIAL_RESULT_COUNT },
+    categories: categoriesResource,
+  });
+
   closeModal = () => {
     this.props.closeCB();
   }
 
+  onSelectRow = (e, contact) => {
+    this.props.addContacts(contact);
```

Comment:
I left a comment in ui-organizations, it would be great to pass array of contacts to the callback already